Eduardosancho develop main page

eduardosancho opened the pull request:
#What is included in this PR

'''
-Create headline with name and personal info
-Add menu icon
-Decorate with some template images
-Organize mobile layout using Flexbox
'''

eduardosancho committed:
Add linters for HTML and CSS

eduardosancho committed:
Create empty base files to start developing webpage

eduardosancho committed:
Add navbar

eduardosancho committed:
Add first section, name and bio

eduardosancho committed:
Create .gitignore to run linter tests manually

eduardosancho committed:
Fix styles.css indentation syntax

eduardosancho committed:
Organize head element in index.html

eduardosancho committed:
Add hover style to "Start collaborating" button

eduardosancho committed:
Erase empty files that were bothering the linters

eduardosancho committed:
Erase hint-report/ from .gitignore

eduardosancho committed:
Add ReadMe with title and description

eduardosancho committed:
Add screenshot to ReadMe.md

eduardosancho committed:
Add github profile to Readme

leolpaz commented on styles/style-main.css:
```diff
@@ -0,0 +1,95 @@
+.navbar {
```
- [x] It'd be better if we followed Figma's given design, I feel that perhaps you got confused, as you created the base for the desktop version
Figma mobile design:
![image](https://user-images.githubusercontent.com/73140461/143499544-50939605-6167-4f7a-aaf8-4abf88ca3864.png)

Yours:
![image](https://user-images.githubusercontent.com/73140461/143501023-5d06055e-1bf1-4f12-857b-8420a587fef4.png)

Just in case you don't know it already, you can change your browser to view as mobile, by right-clicking the page, clicking on inspect, and clicking here: 
![image](https://user-images.githubusercontent.com/73140461/143499654-c0cbd462-7b18-4141-a76f-7a5093a3f876.png)

And then changing up which size do you want here:
![image](https://user-images.githubusercontent.com/73140461/143499684-7e48a6e5-bdd1-4742-8587-1c42d1726375.png)

leolpaz replied:
- [x] When you're now copying the mobile version, please don't forget to add this as a background and position it correctly:
![image](https://user-images.githubusercontent.com/73140461/143500220-c76ccdbc-0538-4eee-b4a5-80f33bc3f6a7.png)

leolpaz commented on index.html:
```diff
@@ -0,0 +1,56 @@
+</head>
+
+<body>
+    <nav class="navbar">
+        <ul>
+            <li><a href="#">Hello</a></li>
+            <li><a href="#">Portfolio</a></li>
+            <li><a href="#">About</a></li>
+            <li><a href="#">Contact</a></li>
+        </ul>
+    </nav>
```
- [x] Kindly wrap this entire `<nav></nav>` inside a `<header></header>`

leolpaz commented on index.html:
```diff
@@ -0,0 +1,56 @@
+
+<body>
+    <nav class="navbar">
+        <ul>
+            <li><a href="#">Hello</a></li>
+            <li><a href="#">Portfolio</a></li>
+            <li><a href="#">About</a></li>
+            <li><a href="#">Contact</a></li>
+        </ul>
```
- [x] Kindly change this `ul` and all of its subItems, to simply be the hamburger menu icon
![image](https://user-images.githubusercontent.com/73140461/143499908-c22be4ee-1540-4c87-bf82-9fbb23c68fd4.png)

leolpaz commented on index.html:
```diff
@@ -0,0 +1,56 @@
+            <div class="headline-left">
+
+                <h1 class="headline-name">
+                    Charlie <br>
+                    Giorgania
```
- [x] Kindly change this from 'Charlie Giorgania" to your own name, as it's going to be your personal portfolio

leolpaz commented on index.html:
```diff
@@ -0,0 +1,56 @@
+
+            </div>
+
+            <div class="headline-right">
+                <img id="shapes" src="/images/image-geometry_1.png" alt="beautiful colorful shapes">
+            </div>
```
- [x] As per the design, kindly delete this image and use it as a background instead.
Hint: You can create an empty div, and apply this image as background, to ease positioning

leolpaz commented on styles/style-main.css:
```diff
@@ -0,0 +1,95 @@
+}
+
+.headline-name {
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
```
- [x] Kindly use the correct font family, the one for this would be the [Playfair Display, which you can find on google fonts](https://fonts.google.com/specimen/Playfair+Display)

leolpaz commented on styles/style-main.css:
```diff
@@ -0,0 +1,95 @@
+}
+
+.headline-bio {
+  font-family: 'Poppins', sans-serif;
```
- [x] While you got the correct font family here, the link on your `<head></head>` shows me that you got Poppins light, kindly change it to Poppins regular. Since the font is provided via web resource, changing the font-weight as you do in the line below without having the one with correct weight on the `<head></head>` doesn't achieve the same effect

leolpaz commented on styles/style-main.css:
```diff
@@ -0,0 +1,95 @@
+#start-collaboration {
+  background-color: rgb(14, 14, 68);
+  color: white;
+  font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
```
- [x] The font family here is also Poppins, but with a weight of 500, so it'd be better if you also changed it
![image](https://user-images.githubusercontent.com/73140461/143500718-29f03c6e-77fc-4098-aee8-d9d89015ee61.png)

leolpaz reviewed (changes requested):
Hi @eduardosancho , 

You've done an amazing job so far, I'd like to highlight the following:
- Repo looks professional
- No Linter errors

Some issues need to be resolved before your approval, but you're almost there!

![](https://media2.giphy.com/media/Jq824R93JsLwZCaiSL/giphy.gif)

## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

eduardosancho committed:
Wrap navbar into <header>

eduardosancho committed:
Add background image, upper-left corner

eduardosancho committed:
Change image_geometry__1 to image_geometry_2

eduardosancho committed:
Rename headline-right to headline-last in css file

eduardosancho committed:
Remove dots image

eduardosancho committed:
Downlaod template .png to images folder

eduardosancho committed:
Erase all flex related styling for the header

eduardosancho committed:
Change headline-name to Eduardo

eduardosancho committed:
Erase Poppins font import data

eduardosancho committed:
Add Poppins regular to headline-bio

eduardosancho committed:
Add Poppins font to Collab Button

eduardosancho committed:
Add Playfair-Display-Font to headline.name

eduardosancho committed:
Remove unnecessary margin-left from .headline sect

eduardosancho committed:
Add hamburger icon for Menu and hide navbar items

eduardosancho committed:
Add font weight to headline name

eduardosancho committed:
Rename headline-left into headline-top

eduardosancho committed:
Add flex style to headline section for mobile

eduardosancho committed:
Change navbar menu icon from text to png

eduardosancho committed:
Fix linter alerts

eduardosancho committed:
Change this > *  for a specific tag in styles.css

julie-ify commented on index.html:
```diff
@@ -0,0 +1,62 @@
+
+            </div>
+
+            <div class="headline-last">
+                <img id="shapes" src="/images/image_geometry_2.png" alt="beautiful colorful shapes">
+            </div>
```
- [x] Please consider deleting this `img` tag and applying the image as a background image in your `CSS` as was requested by the previous reviewer.

> As per the design, kindly delete this image and use it as a background instead.
Hint: You can create an empty div, and apply this image as background, to ease positioning

julie-ify commented on styles/style-main.css:
```diff
@@ -0,0 +1,103 @@
+.navbar ul {
+  display: none;
+
+  /* SAVED CONFIG FOR DROP DOWN MENU ITEMS
+  display: flex;
+  flex-direction: column;
+  align-self: center;
+  align-items: center;
+  padding-inline-start: 0px;
+  */
+}
```
- [x] I can see you want to save this piece of code for future use 👍. However, let's imagine a recruiter inspecting your work right now and seeing these commented lines of code would not make your work look professional. I think it would be better to move them to your notepad or sticky notes and save them there then you can get them when you need to make use of them in your work 😃

julie-ify reviewed (changes requested):
## Hi @eduardosancho, 

Good job so far!

Highlight! 
- Good code organization ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

eduardosancho committed:
Add useful_snippets.txt to .gitignore

eduardosancho committed:
Set backgrounds on header and move navbar styling to useful_snippets.txt

They were setup as regular images, to change that i did the following:
-Create an empty div with an id tag for each background images
-Use that ID to give style to position the backgrounds

The navbar is hidden for now, I wanted to keep the styling as a comment,
but instead I copied it into useful_snippets.txt

eduardosancho committed:
Add empty divs for previous commit "backgrounds"

eduardosancho committed:
Make headline-list background grow and shrink

eduardosancho committed:
Fix headline-bio text

eduardosancho committed:
Linter fix on style-main.css

brytebee reviewed (approved):
Hi @eduardosancho  👋,

# STATUS: APPROVED 🕺🥳 

Congratulations🍾🥳!! Your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 

![](https://media.giphy.com/media/fxsqOYnIMEefC/giphy.gif)

- [ ] **Optional** Kindly update the font-weight of the `healine-name` to 700

Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
